README.md

```diff
@@ -155,26 +154,36 @@ python3 benchmark.py \
-| Google Speech-to-Text (Enhanced) |         6.62%          |         13.59%         |  6.68%   |   18.39%    | 11.32%  |
-|    IBM Watson Speech-to-Text     |         11.08%         |         26.38%         |  11.89%  |   38.81%    | 22.04%  |
-|        Mozilla DeepSpeech        |         7.27%          |         21.45%         |  18.90%  |   43.82%    | 22.86%  |
-|        Picovoice Cheetah         |         7.08%          |         16.28%         |  10.89%  |   23.10%    | 14.34%  |
-|        Picovoice Leopard         |         5.39%          |         12.45%         |  9.04%   |   17.13%    | 11.00%  |
-
-### RTF
-
-Measurement is carried on an Ubuntu 20.04 machine with Intel CPU (`Intel(R) Core(TM) i5-6500 CPU @ 3.20GHz`), 64 GB of
-RAM, and NVMe storage.
-
-|       Engine       | RTF  | Model Size |
-|:------------------:|:----:|:----------:|
-| Mozilla DeepSpeech | 0.46 |  1142 MB   |
-| Picovoice Cheetah  | 0.07 |   19 MB    |
-| Picovoice Leopard  | 0.05 |   19 MB    |
+![](results/plots/WER.png)
+
+|             Engine             | LibriSpeech test-clean | LibriSpeech test-other | TED-LIUM | CommonVoice | Average |
+|:------------------------------:|:----------------------:|:----------------------:|:--------:|:-----------:|:-------:|
+|       Amazon Transcribe        |          2.6%          |          5.6%          |   3.8%   |    8.7%     |  5.2%   |
+|      Azure Speech-to-Text      |          2.8%          |          6.2%          |   4.6%   |    8.9%     |  5.6%   |
+|     Google Speech-to-Text      |         10.8%          |         24.5%          |  14.4%   |    31.9%    |  20.4%  |
+| Google Speech-to-Text Enhanced |          6.2%          |         13.0%          |   6.1%   |    18.2%    |  10.9%  |
+|   IBM Watson Speech-to-Text    |         10.9%          |         26.2%          |  11.7%   |    39.4%    |  22.0%  |
+|  Whisper Large (Multilingual)  |          3.7%          |          5.4%          |   4.6%   |    9.0%     |  5.7%   |
+|         Whisper Medium         |          3.3%          |          6.2%          |   4.6%   |    10.2%    |  6.1%   |
+|         Whisper Small          |          3.3%          |          7.2%          |   4.8%   |    12.7%    |  7.0%   |
+|          Whisper Base          |          4.3%          |         10.4%          |   5.4%   |    17.9%    |  9.5%   |
+|          Whisper Tiny          |          5.9%          |         13.8%          |   6.5%   |    24.4%    |  12.7%  |
+|       Picovoice Cheetah        |          5.6%          |         12.1%          |   7.7%   |    17.5%    |  10.7%  |
+|       Picovoice Leopard        |          5.3%          |         11.3%          |   7.2%   |    16.2%    |  10.0%  |
+
+### Core-Hour & Model Size
+
+To obtain these results, we ran the benchmark across the entire TED-LIUM dataset and recorded the processing time.
+The measurement is carried out on an Ubuntu 22.04 machine with AMD CPU (`AMD Ryzen 9 5900X (12) @ 3.70GHz`),
```

Comment:
Elsewhere in this readme we say that this benchmark was `developed and tested on Ubuntu 20.04.`. Probably should change that

Reply:
Changed it!